suisa_sendemeldung/suisa_sendemeldung.py

```diff
@@ -585,8 +616,8 @@ def get_csv(data: dict, args: ArgparseNamespace) -> str:
         title = music.get("title")
 
         artist = get_artist(music)
+        composer = get_composer(music)
```

Comment:
We might want to catch the error here:


```suggestion
        try:
            composer = get_composer(music)
        catch TypeError:
            composer = ""
```

or the get_composer method could also just only return a string, not sure what is simpler